options/ansi/musl-generic-math/libm.h

```diff
@@ -17,7 +17,7 @@
 #include <float.h>
 #include <math.h>
 //#include <complex.h>
-#include <endian.h>
+// #include <endian.h>
```

Comment:
Don't comment it out, just remove it. Same for the ``complex.h`` include.

Reply:
Done as requested, should be fine now pending CI completion.